assets/js/googlesitekit-module.js

```diff
@@ -37,36 +36,14 @@ import Root from './components/root';
 import ModuleApp from './components/module-app';
 import Setup from './components/setup/ModuleSetup';
```

Comment:
Let's rename this to `ModuleSetup` for consistency while we're at it 🙂 

```suggestion
import ModuleSetup from './components/setup/ModuleSetup';
```